geth/api/backend_jail_test.go

```diff
@@ -249,11 +250,10 @@ func (s *BackendTestSuite) TestContractDeployment() {
 			gas: '` + strconv.Itoa(params.DefaultGas) + `'
 		}, function (e, contract) {
 			// NOTE: The callback will fire twice!
+			errorValue = e;
```

Comment:
A general rule in nodejs is that if a callback has an error, you should not continue executing but return early. In this case, I guess `contract` is always present, but it might not be and it still should work.

```js
if (e) {
  errorValue = e;
  return
}

if (!contract.address) {
  responseValue = contract.transactionHash;
}
```

Reply:
@adambabik where did you see that panic? I can't find it in Travis logs.

Reply:
Hmm. I got this panic as well, it's the test that need to fixed.